oda_api/token.py

```diff
@@ -66,6 +65,34 @@ def decode_oauth2_token(token: str):
     # usually comes in cookies['_oauth2_proxy']
     return json.loads(base64.b64decode(token.split(".")[0]+"=").decode())
 
+
+def rewrite_token(new_token,
+                  token_discovery_methods=(
+                          "environment variable ODA_TOKEN",
+                          "file in current directory",
+                          "file in home")):
+
+    for n in [
+        "environment variable ODA_TOKEN",
```

Comment:
since these modes are now in two places, and even maybe will be exposed to the user, they could be replaced with Enum.